tests/test_gui.py

```diff
@@ -617,14 +697,18 @@ def test_publishes_tab(
     Publishes tab validation
     """
     # Wait for the UI to show up and click on the Publishes tab
-    app_dialog.root.buttons["Click to go to your work area"].waitExist(timeout=30)
-    app_dialog.root.buttons["Click to go to your work area"].mouseClick()
+    # PySide2 doesn't see button's name so we need to use integer to point to it in the hierarchy list
+    if app_dialog.root.buttons["Click to go to your work area"].exists() is True:
```

Comment:
I think that replacing exists with waitExist might be better since we can face some delays with objects creation.

Reply:
waitExist() can't be used with if condition.
But I added back waitExist() at the beginning of each test to make sure UI is up before starting the test.